### DISCUSSION TITLE
커밋 컨벤션

### DISCUSSION THREAD

**Discussion by codeisneverodd** [prgrms-fe-devcourse/FEDC2_TimeDivider_Root#3] [1. 비개발]
prefix | 설명 | 추가 설명
-- | -- | --
feat | 기능추가/수정/삭제 | 제품 코드 수정 발생
refactor | 리팩토링 | 제품 코드 수정 발생
fix | 버그 수정 | 제품 코드 수정 발생
docs | 문서 추가, 수정, 삭제 | 코드 수정 없음
etc | 그 외 모든 수정 | 유형이 혼재된 경우, 되도록이면 커밋 분리. 분리가 어려울 경우 위 순서 상 상위 항목의 유형으로 작성

**Comment by ljw0096** [comment]
ex) feat: 지원 커밋

**Comment by dahye1013** [comment]
chore : 그 외 자잘한 수정에 대한 커밋
 ci : CI 관련 설정 수정에 대한 커밋 
 style : ui 스타일에 관한 커밋 
 refactor : 코드 리팩토링에 대한 커밋 
 release: 릴리즈에 대한 커밋
이 정도는 좀 더 추가되면 좋을 것 같아요!

**Reply by codeisneverodd** [reply]
etc => chore
ci 불필요
style 추가
refactor는 있습니다!
release 도 추가

**Reply by dahye1013** [reply]
좋아요~~!

**Reply by codeisneverodd** [reply]
prefix | 설명 | 추가 설명
-- | -- | --
feat | 기능추가/수정/삭제 | 제품 코드 수정 발생
refactor | 리팩토링 | 제품 코드 수정 발생
fix | 버그 수정 | 제품 코드 수정 발생
docs | 문서 추가, 수정, 삭제 | 코드 수정 없음
style | 스타일 수정 | 제품 코드 수정 발생
chore | 그 외 모든 수정 | 유형이 혼재된 경우, 되도록이면 커밋 분리. 분리가 어려울 경우 위 순서 상 상위 항목의 유형으로 작성

**Reply by codeisneverodd** [reply]
지원님이 : 뒤에 공백 꼭 지켜주셔야한다고 합니다.

**Reply by dahye1013** [reply]
release는 deploy용 브런치 커밋으로 쓰는건 어떨까요? 배포 환경이 조금 씩 바뀔수도 있으니까요!

**Reply by dahye1013** [reply]
미사용파일 제거용 커밋 remove도 추가 부탁드립니답..!
